### DISCUSSION TITLE
Problem with editing resources

### DISCUSSION THREAD

**Discussion by martio** [laravel/nova-issues#3212] [Q&A] [answered]
- Laravel Version: 8.25.0
- Nova Version: 3.20.0
- PHP Version: 8.0.0

### Description:

Problem with editing resources when the value is "user":

```
            Select::make(__('nova.resources.fields.user.role'), 'role')
                ->options([
                    'user' => __('nova.resources.options.user.role.user'),
                    'master' => __('nova.resources.options.user.role.master'),
                    'admin' => __('nova.resources.options.user.role.admin'),
                ])
                ->rules('required', 'in:user,master,admin')
                ->displayUsingLabels()
                ->hideFromIndex(),
```
<img width="1440" alt="Zrzut ekranu 2021-01-31 o 19 04 08" src="https://user-images.githubusercontent.com/133636/106393398-28f2ba00-63f7-11eb-84cf-51eb5e43c733.png">

**Comment by crynobone** [comment, marked as answer]
`Data truncated for column "role"` usually occurred when you trying to update a column with less length than given data. Can you share the migration for adding `role`?

**Reply by martio** [reply]
Sorry, it's okay. My mistake...
